assets/promptflow/models/rai-eval-ui-dag-flow/spec.yaml

```diff
@@ -0,0 +1,12 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: rai-eval-ui-dag-flow
+path: ./
+properties:
+  is-promptflow: true
+  azureml.promptflow.section: gallery
+  azureml.promptflow.type: evaluate
+  azureml.promptflow.name: QnA Quality & Safety Evaluation
```

Comment:
this is the same name as the old flow rai-qna-quality-safety-eval, and it may cause some pipeline failing. System content team may complain.

Reply:
changed to a different name